src/shared/components/case-editor/services/page-validation.service.ts

```diff
@@ -8,31 +8,35 @@ import { WizardPage } from '../domain/wizard-page.model';
 
 @Injectable()
 export class PageValidationService {
-  constructor(private caseFieldService: CaseFieldService) {}
+  constructor(private caseFieldService: CaseFieldService) {
+  }
 
   isPageValid(page: WizardPage, editForm: FormGroup): boolean {
     return page.case_fields
       .filter(caseField => !this.caseFieldService.isReadOnly(caseField))
-      .filter(caseField => !this.isHidden(caseField, editForm))
+      .filter(caseField => this.isHidden(caseField, editForm))
       .every(caseField => {
         let theControl = editForm.controls['data'].get(caseField.id);
         return this.checkDocumentField(caseField, theControl) && this.checkOptionalField(caseField, theControl);
       });
   }
 
+  public isHidden(caseField: CaseField, editForm: FormGroup): boolean {
```

Comment:
This is _not_ indicating when a field is hidden - if it matches the `show_condition` it is, by definition, shown. Can you rename this method, please, or else it'll be confusing... especially seeing as you've flipped the logic on line 17 so it's apparently now filtering out _shown_ fields.

Reply:
I will revert the logic since it won't make sense in other contexts the method is being used. Thanks for pointing this out.